UnitsNet.Tests/CustomCode/TorqueTests.cs

```diff
@@ -49,6 +49,8 @@ public class TorqueTests : TorqueTestsBase
 
         protected override double TonneForceMillimetersInOneNewtonMeter => 1.01972e-1;
 
+        protected override double PoundalFeetInOneNewtonMeter => 23.730360404231938;
```

Comment:
Verified https://www.google.com/search?sxsrf=ALeKk021uyzz2lY7xp-4PkOig305bWestg%3A1602702638134&ei=Lk2HX5bcB4TxrgTvuaeACg&q=PoundalFoot+In+One+NewtonMeter&oq=PoundalFoot+In+One+NewtonMeter&gs_lcp=CgZwc3ktYWIQAzoECCMQJ1D9DljcMGCbMmgBcAB4AIABoAKIAdILkgEFMC44LjGYAQCgAQGqAQdnd3Mtd2l6wAEB&sclient=psy-ab&ved=0ahUKEwiWq7TL5LTsAhWEuIsKHe_cCaAQ4dUDCA0&uact=5